Highway.h

```diff
@@ -2,7 +2,9 @@
 
 #include <vector>
 #include <algorithm>
+#include "Vehicle.h"
 
+struct HighwayPatrol;
 struct Highway
```

Comment:
Don't include what you don't use. 
If you're not using anything from `Vehicle`, forward-declare it, and include the header in the cpp file
- [ ] revise